sdk/servicebus/service-bus/src/models.ts

```diff
@@ -22,8 +22,15 @@ export interface ProcessErrorArgs {
    * 'processMessageCallback': Errors thrown from the user's `processMessage` callback passed to `subscribe`.
    * 'receive': Errors thrown when receiving messages.
    * 'renewLock': Errors thrown when automatic lock renewal fails.
-   */
-  errorSource: "abandon" | "complete" | "processMessageCallback" | "receive" | "renewLock";
+   * 'internal': Errors thrown when the library fails internally.
```

Comment:
What do you think of adding a `This should not happen. If it does, please log an issue at .... with the error details`?

Reply:
If we end up resurrecting it I like that idea. I ended up removing it because, I realized, it was a silly catch.